tests/acl/test_acl_outer_vlan.py

```diff
@@ -182,6 +199,26 @@ def setup_vlan(rand_selected_dut, vlan_setup_info):
     rand_selected_dut.shell_cmds(cmds=cmds)
     time.sleep(10)
 
+    # Create portchannel
+    # Port in acl table can't be added to port channel, and acl table can only be updated by json file
+    duthost.remove_acl_table("EVERFLOW")
+    duthost.remove_acl_table("EVERFLOWV6")
+    rand_selected_dut.shell_cmds(cmds=["config portchannel add {}".format(DUT_LAG_NAME)])
+    cmds = []
+    for port_name in portchannel_setup[DUT_LAG_NAME]['member']:
+        cmds.append("config portchannel member add {} {}".format(DUT_LAG_NAME, port_name))
+    rand_selected_dut.shell_cmds(cmds=cmds)
+    time.sleep(10)
```

Comment:
>sleep [](http://example.com/codeflow?start=9&length=5)

Could you wait an event? It will be more robust and efficient than waiting a fixed time.

Reply:
updated.